x-pack/plugins/gis/public/shared/layers/tile_layer.js

```diff
@@ -13,6 +13,7 @@ import { TileStyle } from '../layers/styles/tile_style';
 export class TileLayer extends ALayer {
 
   static type = "TILE";
+  TMS_LOAD_TIMEOUT = 32000;
```

Comment:
Where did 32 seconds come from?

Reply:
Offline discussion with @thomasneirynck but for resolution of the TMS layers in the legacy map apps. For consistency, I made the timeout the same here